From 9511b412b752f11b4e91293a3e2c7f908cda0e68 Mon Sep 17 00:00:00 2001
From: Xiaokang Qin <xiaokang.qin@intel.com>
Date: Tue, 10 Sep 2013 12:58:09 +0800
Subject: REVERTME: HACK restore old log macros

Some old native projects use the pre-JB liblog macros. Temprorarily
restore them until they can be updated to use the new APIs.

JIRA Link to track cleanup so we can eventually remove this:
https://otci.intel.com/jira/browse/ADEV-138

Bug: 46708
Category: Device Enablement
Domain: AOSP-liblog
Severity: High
Origin: Internal
Upstream-candidate: No, this is a hack that needs to be removed

Change-Id: Ieb34d7b1daae51b7e54797c249e887dd08814a2e
Signed-off-by: Xiaokang Qin <xiaokang.qin@intel.com>
---
 include/log/log.h |   30 +++++++++++++++++++++++++++++-
 1 files changed, 29 insertions(+), 1 deletions(-)

diff --git a/include/log/log.h b/include/log/log.h
index 7faddea..a41ea5a 100644
--- a/include/log/log.h
+++ b/include/log/log.h
@@ -67,6 +67,9 @@ extern "C" {
 #ifndef LOG_TAG
 #define LOG_TAG NULL
 #endif
+//////
+#define LOGD ALOGD
+#define LOGD_IF ALOGD_IF
 
 // ---------------------------------------------------------------------
 
@@ -80,6 +83,8 @@ extern "C" {
 #define ALOGV(...) ((void)ALOG(LOG_VERBOSE, LOG_TAG, __VA_ARGS__))
 #endif
 #endif
+/////
+#define LOGV ALOGV
 
 #define CONDITION(cond)     (__builtin_expect((cond)!=0, 0))
 
@@ -93,6 +98,8 @@ extern "C" {
     : (void)0 )
 #endif
 #endif
+//////
+#define LOGV_IF ALOGV_IF
 
 /*
  * Simplified macro to send a debug log message using the current LOG_TAG.
@@ -107,6 +114,9 @@ extern "C" {
     ? ((void)ALOG(LOG_DEBUG, LOG_TAG, __VA_ARGS__)) \
     : (void)0 )
 #endif
+//////
+#define LOGD ALOGD
+#define LOGD_IF ALOGD_IF
 
 /*
  * Simplified macro to send an info log message using the current LOG_TAG.
@@ -121,6 +131,9 @@ extern "C" {
     ? ((void)ALOG(LOG_INFO, LOG_TAG, __VA_ARGS__)) \
     : (void)0 )
 #endif
+///////
+#define LOGI ALOGI
+#define LOGI_IF ALOGI_IF
 
 /*
  * Simplified macro to send a warning log message using the current LOG_TAG.
@@ -135,6 +148,9 @@ extern "C" {
     ? ((void)ALOG(LOG_WARN, LOG_TAG, __VA_ARGS__)) \
     : (void)0 )
 #endif
+//////
+#define LOGW ALOGW
+#define LOGW_IF ALOGW_IF
 
 /*
  * Simplified macro to send an error log message using the current LOG_TAG.
@@ -149,6 +165,9 @@ extern "C" {
     ? ((void)ALOG(LOG_ERROR, LOG_TAG, __VA_ARGS__)) \
     : (void)0 )
 #endif
+///////
+#define LOGE ALOGE
+#define LOGE_IF ALOGE_IF
 
 // ---------------------------------------------------------------------
 
@@ -163,6 +182,8 @@ extern "C" {
 #define IF_ALOGV() IF_ALOG(LOG_VERBOSE, LOG_TAG)
 #endif
 #endif
+/////
+#define IF_LOGV IF_LOGV
 
 /*
  * Conditional based on whether the current LOG_TAG is enabled at
@@ -171,6 +192,8 @@ extern "C" {
 #ifndef IF_ALOGD
 #define IF_ALOGD() IF_ALOG(LOG_DEBUG, LOG_TAG)
 #endif
+/////
+#define IF_LOGD IF_ALOGD
 
 /*
  * Conditional based on whether the current LOG_TAG is enabled at
@@ -179,6 +202,8 @@ extern "C" {
 #ifndef IF_ALOGI
 #define IF_ALOGI() IF_ALOG(LOG_INFO, LOG_TAG)
 #endif
+////
+#define IF_LOGI IF_ALOGI
 
 /*
  * Conditional based on whether the current LOG_TAG is enabled at
@@ -187,6 +212,8 @@ extern "C" {
 #ifndef IF_ALOGW
 #define IF_ALOGW() IF_ALOG(LOG_WARN, LOG_TAG)
 #endif
+/////
+#define IF_LOGW IF_ALOGW
 
 /*
  * Conditional based on whether the current LOG_TAG is enabled at
@@ -195,7 +222,8 @@ extern "C" {
 #ifndef IF_ALOGE
 #define IF_ALOGE() IF_ALOG(LOG_ERROR, LOG_TAG)
 #endif
-
+////
+#define IF_LOGE IF_ALOGE
 
 // ---------------------------------------------------------------------
 
-- 
1.7.4.1

